src/main/kotlin/org/wfanet/panelmatch/client/storage/SharedStorageSelector.kt

```diff
@@ -64,7 +64,9 @@ class SharedStorageSelector(
       GOOGLE_CLOUD_STORAGE -> require(platform == PlatformCase.GCS)
       AMAZON_S3 -> require(platform == PlatformCase.AWS)
       StorageType.UNKNOWN_STORAGE_CLIENT, StorageType.UNRECOGNIZED ->
-        throw IllegalArgumentException("$storageType unsupported")
+        require(platform !in setOf(PlatformCase.GCS, PlatformCase.AWS))
+    // TODO(@efoxepstein): throw IllegalArgumentException("$storageType unsupported")
```

Comment:
https://github.com/world-federation-of-advertisers/cross-media-measurement-api/pull/73